cards/cards.css

```diff
@@ -1,161 +1,172 @@
-  }
-  
-  .close-btn:hover {
-    background: var(--danger-color);
-  }
-  
-  .figure-with-dismiss {
-    position: relative;
-  }
-  
-
-  .box-with-shadow {
-    box-shadow: 1px 1px 1px 1px;
-  }
\ No newline at end of file
+  max-inline-size: 100%;
+  block-size: auto;
+  border-radius: 1rem 1rem 0rem 0rem;
+}
+
+.card {
+  background-color: var(--secondary-color);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  flex-wrap: wrap;
+  width: fit-content;
+  max-width: 350px;
+  height: min-content;
+  border-radius: 1rem;
+  position: relative;
+}
+
+.card:hover {
+  transition: 0.5sec;
+  box-shadow: 0 0 1rem 0.1rem var(--secondary-color);
+}
+
+.card-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card-body:hover {
+  background-color: var(--hover-colour);
+  cursor: pointer;
+  border-radius: 1rem 1rem 0rem 0rem;
+}
+
+.text-container {
+  padding: 1rem 1rem 0.2rem 1rem;
+  flex: 1 1 auto;
+  color: var(--secondary-text-color);
+}
+
+.title-text {
+  font-size: 1.25rem;
+  color: var(--primary-text-color);
+  font-weight: bolder;
+}
+
+.author-text {
+  font-weight: bold;
+  padding: 0.25rem 0rem 0rem 0rem;
+}
+
+.description {
+  margin: 0.5rem 0rem 0.5rem 0;
+  padding: 0rem 1rem 0rem 1rem;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  color: var(--secondary-text-color);
+}
+
+.foot {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  flex: 1 1 auto;
+  gap: 0.2rem;
+  justify-content: space-around;
+  padding: 0rem 0.75rem 0.5rem 0.75rem;
+}
+
+.action-buttons {
+  display: flex;
+  flex: 1 1 auto;
+  flex-wrap: wrap;
+  gap: 0.2rem;
+}
+
+.action-icons {
+  display: flex;
+  flex-wrap: nowrap;
+}
+
+.action-icons > * {
+  padding: 0.75rem;
+  border-radius: 50%;
+}
+
+.action-icons > *:hover {
+  background-color: var(--hover-colour);
+  cursor: pointer;
+}
+
+.basic-textovermedia > .card-body > .figure {
+  position: relative;
+}
+
+.basic-textovermedia > .card-body > .figure > .text-container {
+  z-index: 1;
+  bottom: 0;
+  position: absolute;
+  color: white;
+}
+
+.basic-with-header > .card-body > img {
+  border-radius: 0;
+}
+
+.basic-only-buttons .action-icons {
+  display: none;
+}
+
+.basic-only-icons .action-buttons {
+  display: none;
+}
+
+.horizontal > .card-body > .img-text > img {
+  width: 100px;
+  height: 90px;
+  flex: 0 1 auto;
+  border-top-right-radius: 0;
+}
+
+.horizontal > .card-body > .img-text {
+  display: flex;
+}
+
+.horizontal > .description {
+  display: none;
+}
+
+.card .badge {
+  background-color: var(--primary-color);
+  position: absolute;
+  font-size: 0.75rem;
+  display: flex;
+  top: 0.5rem;
+  left: 0.5rem;
+  justify-content: center;
+  align-items: center;
+  border-radius: 0.5rem;
+  padding: 0.3rem;
+  color: white;
+}
+
+.figure-with-badge {
+  position: relative;
+}
+
+.close-btn {
+  position: absolute;
+  font-size: 0.75rem;
+  display: flex;
+  top: 0.3rem;
+  right: 0.3rem;
```

Comment:
For such small rem values, you can use px.

Reply:
ok... will do that from now. Thanks.